kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1416,17 +1370,62 @@ private Att addKoreAttributes(Production prod, SetMultimap<KLabel, Rule> functio
         String format = att.getOptional("format").orElse(Formatter.defaultFormat(prod.items().size()));
         int nt = 1;
         boolean hasFormat = true;
+        int terminals = 0;
```

Comment:
I don't see this used anywhere. It's just incremented in the below loop, but then the value doesn't seem to be used.

Reply:
@dwightguth I still don't see any uses of this variable.